apps/test-low-memory.sh

```diff
@@ -11,16 +11,26 @@ if [ -n "$CIRCLECI" ]; then \
 fi && \
 
 node --max_old_space_size=4096 `npm bin`/grunt storybookTest && \
-if [-n "$CIRCLECI" ]; then \
+if [ -n "$CIRCLECI" ]; then \
     bash <(curl -s https://codecov.io/bash) -cF storybook; \
 fi && \
 
 node --max_old_space_size=4096 `npm bin`/grunt scratchTest && \
-if [-n "$CIRCLECI" ]; then \
+if [ -n "$CIRCLECI" ]; then \
     bash <(curl -s https://codecov.io/bash) -cF scratch; \
 fi && \
 
-node --max_old_space_size=4096 `npm bin`/grunt integrationTest && \
+LEVEL_TYPE='maze|turtle|bounce|calc|eval|flappy|studio' node --max_old_space_size=4096 `npm bin`/grunt integrationTest && \
```

Comment:
I'm guessing tests run faster with these grouped, but have we tried doing each of them separately?  I wonder if we'd build much smaller bundles and get some speedup just treating each of these as a different entry point.